### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by aminmiri**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -8,14 +8,14 @@ IMG=${IMG:-'quay.io/rogreen/bookmark-generator:latest'}
...
-${CONTAINER_ENGINE} run \
+$CONTAINER_ENGINE run \
     --rm \
     --interactive \
     --tty \
     --volume \
-        "$(pwd)":${CONTAINER_BASE_DIR}/data \
-    "${ADDITIONAL_CONTAINER_ENGINE_ARGS}" \
+    "$(pwd)":$CONTAINER_BASE_DIR/data \
+    $ADDITIONAL_CONTAINER_ENGINE_ARGS \
```
Comment (georgettica) [inline run.sh:17]: Why is removing the quotes the solution?
Not that I'm against it, but curious

Comment (aminmiri) [reply run.sh:17]: double quote in linux always execute what's inside it, so if you try `$"ADDITIONAL_CONTAINER_ENGINE_ARGS"` you will get this error: `ADDITIONAL_CONTAINER_ENGINE_ARGS: command not found`
So whenever you want to call a variable in bash you just add $ before it's name.

Comment (georgettica) [reply run.sh:17]: What about "$ADDITIONAL_CONTAINER_ENGINE_ARGS"
Would that also work?

Comment (aminmiri) [reply run.sh:17]: @georgettica  unfortunately no.
Please if you have confusion about the `"` , `'` and `$` in bash check the answer in this link. It does really provide a good answer for different cases. :D

Comment (georgettica) [reply run.sh:17]: Anyways it's not a big deal..
I'll merge

**Pull request merged**